apps/web/pages/auth/login.tsx

```diff
@@ -174,6 +186,7 @@ export default function Login({
           "--cal-brand-subtle": "#9CA3AF",
         } as CSSProperties
       }>
+      <LoadingBar color="#f11946" progress={progress} onLoaderFinished={() => setProgress(0)} />
```

Comment:
I think we don't want to add a separate loading bar. How about we move the button to loading state(by setting loading=true)Once the navigation completes, the button itself would go away.

This would give a clear indication to the user that the action is being taken/.
cc @sean-brydon